apps/cancel-request/fields/index.js

```diff
@@ -261,8 +261,6 @@ module.exports = {
     validate: [
       'required',
       'notUrl',
-      { type: 'minlength', arguments: 8 },
```

Comment:
I appreciate that the regex will take care of min and max length. However, I don't see a harm leaving min and max validation here. In case we have a separate errors in for min or max length in the future, we can simply update the error message without needing to update the actual logic or the additional validation.

Reply:
You might have not noticed this because the error message are exactly the same (I raised this question [here](https://collaboration.homeoffice.gov.uk/jira/browse/ROD-63). If we can just update the error message for testing purpose, we should see the length error before the actual regex as they run in order.

![Screenshot 2024-10-24 at 15 53 43](https://github.com/user-attachments/assets/bdb74142-f106-4864-98c6-9e25ebcd2664)